README.md

```diff
@@ -26,6 +26,7 @@ Usage of cluster-proportional-autoscaler:
       --v=0: log level for V logs
       --version[=false]: Print the version and exit.
       --vmodule=: comma-separated list of pattern=N settings for file-filtered logging
+      --nodelabels=: comma-separated list of key=value labels for filtered search of nodes and cpus.
```

Comment:
Could you update this to be consistent with output of `cluster-proportional-autoscaler --help`?

Reply:
Sure. I'll put all in one commit.

Reply:
Done